packages/elements/src/components/ino-chip/ino-chip.scss

```diff
@@ -3,62 +3,61 @@
 @use "@material/chips/styles";
 @use 'base/theme';
 
-$default-chip-color: lighten(theme.color(primary), 36.5%);
-$default-chip-contrast-color: rgba(0, 0, 0, 0.87);
-$default-chip-dark-color: rgba(0, 0, 0, 0.87);
+$n-3: #7275A1;
+$n-7: #EDEFF5;
+$primary: #2C02FF;
+$border-focused: rgba(45, 93, 255, 0.3);
```

Comment:
Aren't these colors part of the `theme.scss` or `new-theme.scss`? We should use the imported values then